targets/targets.json

```diff
@@ -4282,7 +4282,8 @@
         "detect_code": ["0823"],
         "device_has_add": ["ANALOGOUT", "CAN", "CRC", "SERIAL_ASYNCH", "SERIAL_FC", "TRNG", "FLASH"],
         "release_versions": ["2", "5"],
-        "device_name": "STM32L496ZG"
+        "device_name": "STM32L496ZG",
+        "bootloader_supported": true
```

Comment:
Could you add bootloader_supported to DISCO_L496AG also ?

Reply:
done